tests/unit_tests/data_steward/gcloud/gcs/gcs_test.py

```diff
@@ -36,22 +36,28 @@ def setUp(self):
         self.file_name: str = 'foo_file.csv'
         self.hpo_id = 'fake_hpo_id'
 
+    @patch('gcloud.gcs.StorageClient._get_hpo_bucket_id')
     @patch('gcloud.gcs.os.environ.get')
-    def test_get_hpo_bucket_not_set(self, mock_environ_get):
+    def test_get_hpo_bucket_not_set(self, mock_environ_get,
+                                    mock_get_hpo_bucket_id):
+
         mock_environ_get.side_effect = [None, '', 'None']
         expected_message = lambda bucket: f"Bucket '{bucket}' for hpo '{self.hpo_id}' is unset/empty"
 
         # run without setting env var (unset env_var)
+        mock_get_hpo_bucket_id.return_value = None
```

Comment:
You could potentially only replace the mocked function and it should work as before 
```suggestion
        mock_get_hpo_bucket_id.side_effect = [None, '', 'None']
        expected_message = lambda bucket: f"Bucket '{bucket}' for hpo '{self.hpo_id}' is unset/empty"

        # run without setting env var (unset env_var)
```